copter/source/docs/terrain-following-manual-modes.rst

```diff
@@ -16,7 +16,7 @@ This page describes the Surface Following setup .  For autonomous modes using do
 Setup and Configuration
 -----------------------
 
-- Connect a downward facing :ref:`lidar or sonar <common-rangefinder-landingpage>` to the vehicle
+- Connect a downward facing rangefinder(for floor/ground tracking) and/or an upward facing rangefinder (for ceiling tracking) :ref:`lidar or sonar <common-rangefinder-landingpage>` to the vehicle. Which is being used is configured by the :ref:`SURFTRAK_MODE<SURFTRAK_MODE>` parameter.
```

Comment:
I think we should separate out, "Which is being used is configured by the :ref:`SURFTRAK_MODE<SURFTRAK_MODE>` parameter." to its own bullet point.  There are actually two ways to select which is being used.  The parameter can be used but there's also an auxiliary switch option 75 ("Surface Tracking Up/Down")